docker-compose.yml

```diff
@@ -27,6 +27,35 @@ services:
       - ./pgadmin-data:/var/lib/pgadmin
     depends_on:
       - db
+  nestjs:
+    build:
+      context: .
+      dockerfile: ./Dockerfile
+    image: tusharchy/nest-and-postgres-application:latest
+    environment:
+      - DB_TYPE=${DB_TYPE}
+      - DB_HOST=${DB_HOST}
+      - DB_PORT=${DB_PORT}
+      - DB_USERNAME=${DB_USERNAME}
+      - DB_PASSWORD=${DB_PASSWORD}
+      - DB_NAME=${DB_NAME}
+      - POSTGRES_SYNC=false
+      - JWT_SECRET=${JWT_SECRET}
+      - PGADMIN_DEFAULT_EMAIL=${PGADMIN_DEFAULT_EMAIL}
```

Comment:
The nest app won't need the PG ADMIN keys I believe.

Reply:
You believe well